.github/cliff.toml

```diff
@@ -20,7 +20,7 @@ body = """
     {% for group, commits in commits | group_by(attribute="group") %}
         #### {{ group | upper_first }}
         {% for commit in commits %}
-            - {{ commit.message | upper_first }} ([{{ commit.id | truncate(length=7, end="") }}]({{ commit.id }}))\
+            - {{ commit.message | upper_first }} ([{{ commit.id | truncate(length=7, end="") }}](https://github.com/vectordotdev/helm-charts/commit/{{ commit.id }}))\
```

Comment:
🤔 just doing the full commit might work?

Ref: https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls#commit-shas

Reply:
Hmm, yeah, that is weird.